### DISCUSSION TITLE
Possible UI changes to entry type inline and card styles.

### DISCUSSION THREAD

**Discussion by iainhenderson** [craftcms/cms#16186] [Ideas] [answered]
Related to my proposal of[ changing the primary click actions for entry type cards](https://github.com/craftcms/cms/discussions/16185). While working that I created a mockup with some other ideas which could work for the entry type displays.

![Frame 3](https://github.com/user-attachments/assets/be9d1249-ba0d-43c6-b9a9-f9a7bb660e2a)

**Points I like**

- Cards have the accordion expand/collapse control for inline cards and the break out control for slide out cards. (top right corners of cards)
- Card design is more consistent for inline vs slide out card  display (consistent header bar and panel background colours)
- The background colour may visually group individual cards better than the coloured borders currently on inline cards. This may do a better job of suggesting context for long inline cards where headers may be off screen.
- The move icon is a little more obvious than the four dots.
- Slight shadows on cards suggest drag behaviour as card are popped off the page.

**Points I have some reservations about**

- Summary images moved to right. This means card information is consistently aligned and more scannable for cards with and without images, but images on right feel a little odd.
- Hidden checkboxes. I like that it reduces the fiddly controls. (especially after adding the accordion and breakout control) I like that it places the checkbox closer to the title entry information so users can more easily select multiple entries, but hiding controls hinders discoverability. Maybe that's OK as bulk edits are less common

**Comment by JayBox325** [comment]
I like these improvements, I'd also suggest that the fields added to the card should have the label.

<img width="852" alt="image" src="https://github.com/user-attachments/assets/f72d00c1-c36d-4ebb-bc9b-5ce07dbb23cf" />

This would ideally be something like:

```
27 Feb Test Product

Shopify ID:
3u4893204923
```

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the feedback! Addressing this in Craft 5.7 via #16983.

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with that change.
